archive-video-award.php

```diff
@@ -0,0 +1,73 @@
+<?php
+get_header();
+?>
+
+<main id="main-content">
+  <section id="archive-video-award">
+    <div class="container">
+      <div class="grid-row">
+
+<?php
+if (have_posts()) {
+
+  while (have_posts()) {
+    the_post();
+    $video_year = get_post_meta($post->ID, '_igv_video_award_video_year', true);
+    $artists = get_post_meta($post->ID, '_igv_video_award_artists', true);
+    $bio = get_post_meta($post->ID, '_igv_video_award_bio', true);
+    $minutes = get_post_meta($post->ID, '_igv_video_award_minutes', true);
+    $seconds = get_post_meta($post->ID, '_igv_video_award_seconds', true);
+
+    if ($wp_query->current_post == 0) {
```

Comment:
some of these meta values are only used for the first video. you should request those within the conditional.